env/work/srw_python/srwl_uti_smp.py

```diff
@@ -0,0 +1,311 @@
+# -*- coding: utf-8 -*-
+#############################################################################
+# SRW Samples library
+# Authors/Contributors: Maksim Rakitin
+# October 26, 2016
+# October 27, 2017
+#############################################################################
+
+import math
+import os
+
+import numpy as np
+from PIL import Image
+from scipy.ndimage.interpolation import rotate
```

Comment:
In the previous version of this module (at least the one I have now), we had these imports only in those functions that used these libraries, e.g. see below. If we put this import in the beginning, without any try-catch, it won't be loadable in the basic configuration, and may not allow for other modules to load. Consider changing it as we had this before.

#-----------------
    def get_image_from_data(self):
        try:
            import numpy as np
        except:
            raise ImportError('The specified npy file cannot be read since {}'.format(self._import_error_msg.format('numpy')))
        data = np.load(self.file_path)
        self.limit_value = 255
        self.data = (data - data.min()) / (data.max() - data.min()) * self.limit_value
        try:
            from PIL import Image
            self.raw_image = Image.fromarray(self.data.astype(np.uint8))
            self.processed_image = self.raw_image
        except:
            print('The processed image will not be saved since {}'.format(self._import_error_msg.format('pillow')))
            self.is_save_images = False

Reply:
Hi Maksim, I wonder would you be able to address this thing - move "imports" to required places and wrap them by try-catch in this your module, srwl_uti_smp.py, or you are busy now? This seems to be the last thing to be sorted-out before updating SRW on GitHub.

Reply:
Yes, I can do it now.

Reply:
Done.